### DISCUSSION TITLE
怎样添加http或socks的代理服务的认证

### DISCUSSION THREAD

**Discussion by cenaia** [libnyanpasu/clash-nyanpasu#799] [Q&A]
尝试在\.config\clash-nyanpasu的config.yaml添加相关内容，重启软件后没有效果
其他设备可直接连接代理
```
# Generated by Clash Nyanpasu
mixed-port: 7890
log-level: info
allow-lan: true
mode: rule
external-controller: 127.0.0.1:9090
secret: ''
ipv6: false
authentication:
- fortg:wd*****
```

**Comment by keiko233** [comment]
请确保在clash字段中开启authentication字段，并在配置文件中加入authentication字段选项

![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/76520798/0fdbe388-1a1b-4705-8b7a-fbd41c6d77f9)

如果担心更新配置字段覆盖可以使用脚本功能处理

**Reply by cenaia** [reply]
尝试了但是没有效果
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/72257969/e264380f-9b2a-4c61-9a65-2f3352830000)
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/72257969/b36ee4da-9da2-4735-885c-ee71cee1451a)

config.yaml
```
# Generated by Clash Nyanpasu
mixed-port: 7890
log-level: info
allow-lan: true
mode: rule
external-controller: 127.0.0.1:9090
secret: ''
ipv6: false
authentication:
- fortg:wd*****
```
